### PR TITLE
made the description more descriptive and added few additional syntax…

### PR DESCRIPTION
**Pull request by WernerGomindes**
… examples

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/colors/terms/rgb/rgb.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-Defines the color value by providing three comma separated values. The first value represents the red value, the second represents the green value, and the third represents the blue value.
+The `rgb()` function in CSS defines a color by specifying its components using three comma-separated values. The first value represents the red component, the second value represents the green component, and the third value represents the blue component. These values can be provided as either numeric values ranging from 0 to 255 or as percentage values ranging from 0% to 100%. By adjusting the values for each component, you can create a wide range of colors, allowing for precise control over the appearance of elements in your web design.
```
Comment (dakshdeepHERE) [inline content/css/concepts/colors/terms/rgb/rgb.md:16]: we avoid first- and second-person pronouns in Docs. So remove use of `you` `we` etc

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Looks good overall but have one comment for you to work on.

**Comment by dakshdeepHERE** [comment]
@WernerGomindes can you commit the changes so we can move forward?

**Comment by SSwiniarski** [comment]
@WernerGomindes It's been several weeks and you haven't addressed the comments, or signed the [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=3853). If we don't see progress in another week we'll need to close this PR as abandoned.

**Comment by SSwiniarski** [comment]
Closing this PR as abandoned.

**Pull request closed without merging**